examples/LiftSim_baseline/A2C/README.md

```diff
@@ -0,0 +1,45 @@
+# LiftSim基线
+
+## 简介
+
+基于PARL库实现A2C算法，应用于[RLSchool][rlschool]库中的电梯调度模拟环境[LiftSim][liftsim]。
+
+## 依赖库
+
++ [paddlepaddle>=1.6.1](https://github.com/PaddlePaddle/Paddle)
++ [parl](https://github.com/PaddlePaddle/PARL)
++ [rlschool>=0.1.1][rlschool]
+
+Windows版本仅支持Python3.5及以上版本。
```

Comment:
Our simulator is not supporting windows.

Reply:
removed